crowdin_api/client.py

```diff
@@ -20,7 +20,7 @@ class CrowdinClient:
     HEADERS = {}
     USER_AGENT = "crowdin-api-client-python"
     PAGE_SIZE = 25
-    HEADERS = {}
+    EXTENDED_PARAMS_REQUEST = None
```

Comment:
I would suggest renaming this

```suggestion
    EXTENDED_REQUEST_PARAMS = None
```

Reply:
Yes, you are right. Changed